routes/changeInventory.js

```diff
@@ -0,0 +1,112 @@
+var express = require("express");
+var Q = require('q');
+
+var router = express.Router();
+
+
+/*
+ Usage:
+ addRun - localhost:50001/addRun/inventoryId/runDate
+ removeRun - localhost:50001/removeRun/inventoryId/runId
+ addBatch - localhost:50001/addBatch/runId/batchAmount/batchLocation
+ removeBatch - localhost:50001/removeBatch/runId/batchAmount/batchLocation
+ Fields:
+ inventoryId, runId, runDate, batchAmount, batchLocation
+ */
+
+// add run
+router.route("/:mode/:inventoryId/:runDate").get(function(req, res) {
+    changeInventory(req, res);
+});
+
```

Comment:
So your usage here does not match what your setup is. To hit add for instance would be:
addRun - localhost:50001/changeInventory/:mode/:inventoryID/:runDate

We should only have one route per file, and if we want "changeInventory" to be a word that needs to be typed in each route, we should have a folder called changeInventory with the different named files inside of it.

Can we set up a time tomorrow before 5 pm that we could either do a hangout or slack session in order to go over this code?